### PR TITLE
Mobile menu js

### PR DESCRIPTION
**Pull request by gfrancoarq**
### I've created a mobile menu on my portfolio using Javascript.
* These changes complete the "Portfolio: mobile menu"

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -167,6 +256,72 @@ nav {
   }
 }
 
+@media only screen and (max-width: 766px) {
```
Comment (prolajumokeoni) [inline style.css:259]: ### OPTIONAL

- [ ] Kindly use the mobile first approach when introducing media queries in your design, please use min-width you can  also use `dispaly:none` to hide elements on larger screen sizes 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @gfrancoarq 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -34,6 +34,7 @@ h1 {
   transition-property: font-size;
   transition-duration: 0.2s;
   transition-delay: 0.1s;
+  animation: appearUp 0.7s ease-in;
 }
```
Comment (ichala) [inline style.css:39]: - [ ]  [OPTIONAL] your website looks fantastic I would love to suggest adding a smoother animation to your website's scroll behavior, try to add `scroll-behavior: smooth;` to your HTML tag in CSS it works like magic! 😮 

``` css
html {
scroll-behavior: smooth;
}
```

Before           | After
:-------------------------:|:-------------------------:
before | mAFTER

**Review by ichala — APPROVED** [review]
Hi @gfrancoarq ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implement✔️
 - Good Design✔️

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


*************************************
# Optional Suggestions 

- [ ] check comments below


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**